sync-todo/v2/client/swiftui/App/realmApp.swift

```diff
@@ -1,7 +1,7 @@
 import SwiftUI
 import RealmSwift
 
-/// This method loads app config details from a Realm.plist we generate
+/// This method loads app config details from a atlasConfig.plist we generate
 /// for the template apps.
 /// When you create your own Realm app, use your preferred method
```

Comment:
Maybe this?
```suggestion
/// When you create your own Atlas Device Sync app, use your preferred method
```

Reply:
I can't comment directly on the line below, but should `realmApp` in ln 10 be renamed & places that use it be refactored to match the new name? (i.e. in this file ln 14, ln 18 - not sure if it's used elsewhere without doing some searching?)